vanilla-framework/vanilla-framework
# Update icons to use background-image instead of mask

## Done

Update icons to use background-image instead of mask

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/patterns/icons/icons-light/
- Open http://0.0.0.0:8101/vanilla-framework/examples/patterns/icons/icons-dark/
- Check icons match [spec](https://github.com/ubuntudesign/vanilla-design/tree/master/Icons)

## Details

Fixes #1346 

FF / IE screenshots

<img width="1554" alt="screen shot 2017-09-20 at 16 27 06" src="https://user-images.githubusercontent.com/505570/30652632-c77a490e-9e20-11e7-9f92-1b663ad1c9fd.png">
<img width="1483" alt="screen shot 2017-09-20 at 16 27 21" src="https://user-images.githubusercontent.com/505570/30652634-c77bba46-9e20-11e7-903e-a11c90094877.png">
<img width="551" alt="screen shot 2017-09-20 at 16 27 42" src="https://user-images.githubusercontent.com/505570/30652635-c780c176-9e20-11e7-917f-ed0b7e2913c4.png">
<img width="558" alt="screen shot 2017-09-20 at 16 27 35" src="https://user-images.githubusercontent.com/505570/30652633-c77ab236-9e20-11e7-9e98-49b93bbf0972.png">
